src/patternfly/components/Tile/tile.scss

```diff
@@ -0,0 +1,170 @@
+  --pf-c-tile--before--m-selected--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled--BackgroundColor: var(--pf-global--disabled-color--300);
+
+  // header
+
+  // title
+  --pf-c-tile__title--Color: var(--pf-global--Color--100);
+  --pf-c-tile--hover__title--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--m-selected__title--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled__title--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-tile__icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-tile__icon--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-tile__icon--Color: var(--pf-global--Color--200);
+  --pf-c-tile--m-selected__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--hover__icon--Color: var(--pf-global--primary-color--100);
+
+  // Stacked variatiion
+  --pf-c-tile__header--m-stacked__icon--MarginBottom: var(--pf-global--spacer--xs);
+  --pf-c-tile__header--m-stacked__icon--FontSize: var(--pf-global--icon--FontSize--lg);
+  --pf-c-tile__header--m-stacked__icon--Color: var(--pf-global--Color--100);
+  --pf-c-tile--hover__header--m-stacked__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--m-selected__header--m-stacked__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled__header--m-stacked__icon--Color: var(--pf-global--Color--200);
+
+  // large variation
+  --pf-c-tile--m-display-lg__header--m-stacked__icon--FontSize: var(--pf-global--icon--FontSize--xl);
+  --pf-c-tile--m-display-lg__header--m-stacked__icon--Width: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--FontSize);
+  --pf-c-tile--m-display-lg__header--m-stacked__icon--Height: var(--pf-c-tile--m-display-lg__header--m-stacked__icon--Width);
+
+  // body
+  --pf-c-tile__body--Color: var(--pf-global--Color--100);
+  --pf-c-tile__body--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-tile--disabled__body--Color: var(--pf-global--disabled-color--100);
+
+  position: relative;
+  display: inline-grid;
+  grid-template-rows: min-content;
+  padding: var(--pf-c-tile--PaddingTop) var(--pf-c-tile--PaddingRight) var(--pf-c-tile--PaddingBottom) var(--pf-c-tile--PaddingLeft);
+  text-align: center;
+  background-color: var(--pf-c-tile--BackgroundColor);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    pointer-events: none;
+    content: "";
+    border: var(--pf-c-tile--before--BorderWidth) solid var(--pf-c-tile--before--BorderColor);
+    border-radius: var(--pf-c-tile--before--BorderRadius);
+  }
+
+  // stylelint-disable
+  &[disabled] {
+    pointer-events: none;
+    cursor: not-allowed;
+
+    --pf-c-tile--BackgroundColor: var(--pf-c-tile--disabled--BackgroundColor);
+    --pf-c-tile__title--Color: var(--pf-c-tile--disabled__title--Color);
+    --pf-c-tile__body--Color: var(--pf-c-tile--disabled__body--Color);
+
+    &::before {
+      border: 0;
```

Comment:
you could just set `--pf-c-tile--before--BorderWidth: 0;` on the `disabled` (or `.pf-m-disabled`) selector